### PR TITLE
fix perl regex to calculate download traffic per day correctly on GHE 2.12.x

### PR DESCRIPTION
**Pull request by rashamalek**
this is tested on GHE 2.12.3, and covers: #98 
Please test it on GHE 2.11.x if you have any running instance.

### CODE REVIEW

File: updater/scripts/git-download.sh [hunk 1]
```diff
@@ -20,15 +20,20 @@ if ghe_greater_equal "2.11.0" ; then
     # most recent log files (because the information from yesterday may or not
     # be rotated already).
     CAT_LOG_FILE="zcat -f /var/log/github-audit.{log.1*,log} | grep -F '$(date --date='yesterday' +'%b %_d')'"
+
+    # The order in github-audit.log in GHE 2.12.x has been changed, to pick the right order PERL_REGEX is created.
+    PERL_REGEX='print if s/.*"cloning":([^,]+).*"program":"upload-pack".*"repo_name":"([^"]+).*"uploaded_bytes":([^,]+).*"user_login":"([^"]+).*/\2\t\4\t\1\t\3/'
```
Comment (larsxschneider) [inline updater/scripts/git-download.sh]: Wouldn't that use the new regex for 2.11, too? The function above checks for `greater_equal`!

Comment (rashamalek) [reply updater/scripts/git-download.sh]: You are right, then I recommend separating the version check for perl regex and the log file into 2 different ifs.

**Comment by larsxschneider** [comment]
Sorry @rashamalek for the delay. Thanks a lot for this fix 👍 - this is awesome 🎉  !

I reformatted your patch a bit and squashed at as 1e2e268f2b31de59e90fc34cde49aaafc38772a0 !

**Comment by rashamalek** [comment]
👍

**Comment by larsxschneider** [comment]
I just realized that you don't show up in the contributors graph. If think you need to add your zalando email address to your github profile to make this work because I squashed the commits and didn't merge the PR directly. Sorry!

**Comment by rashamalek** [comment]
No Worries :), I added the email.

**Comment by pluehne** [comment]
@rashamalek: And thanks for spotting this and submitting the fix 😄!

**Pull request closed without merging**